Cargo.toml

```diff
@@ -82,7 +82,8 @@ git = "https://github.com/pingcap/murmur3.git"
 git = "https://github.com/pingcap/rust-rocksdb.git"
 
 [dependencies.kvproto]
-git = "https://github.com/pingcap/kvproto.git"
+git = "https://github.com/huachaohuang/kvproto.git"
```

Comment:
Before merge will we go back to using the `pingcap` repository?

Reply:
Yes, after I have two LGTMs.